lib/models/storefront.dart

```diff
@@ -0,0 +1,13 @@
+/// Contains the information about the current store account.
+class Storefront {
```

Comment:
Please note how this is not using `freezed`. We wanted to start moving models off freezed, and this seemed like an easy start. Note that this means we won't have some convenience functionality we got with freezed like equals/toString for free... But I think that's ok and we can add that later if needed.

Reply:
yeah equals and toString aren't exactly advanced features lol we can add if we ever see the need